Celeste.Mod.mm/Content/Dialog/English.txt

```diff
@@ -108,6 +112,9 @@
 	MODUPDATECHECKER_INSTALLING=	installing...
 	MODUPDATECHECKER_FAILED=		update failed!
 	MODUPDATECHECKER_WILLRESTART=	press Back to restart Celeste
+	MODUPDATECHECKER_UPDATE_ALL=	Update all mods
+	MODUPDATECHECKER_UPDATE_ALL_INPROGRESS=	Updating all mods...
+	MODUPDATECHECKER_UPDATE_ALL_DONE=	All mods have been updated.
```

Comment:
Uh... this smells like a merge conflict that GitHub is too lazy to spot? We've already got those lines on the master branch.

Please DON'T remove them, as I'm not sure if git will then try to merge the "removal".